siac/walletcmd.go

```diff
@@ -192,8 +192,11 @@ func walletinitcmd() {
 }
 
 // walletload033xcmd loads a v0.3.3.x wallet into the current wallet.
-func walletload033xcmd(source string) {
-	password, err := speakeasy.Ask("Wallet password: ")
+func walletload033xcmd() {
+	fmt.Print("Enter 033x Location: ")
+	var source string
+	fmt.Scanln(&source)
```

Comment:
Can you explain the rationale behind this change a bit more? It doesn't seem like more interactivity necessarily improves clarity here, especially since the confusion came from wallet password vs. seed, not the filepath of the wallet being loaded.

Then again this pattern more closely matches the pattern of `siac wallet load seed`. However doing so gets rid of bash completion which to me is a deal breaker.